custom_components/yandex_maps/sensor.py

```diff
@@ -71,11 +80,12 @@ async def async_update(self):
                     assert resp.status == 200
                     info = await resp.json()
 
-                    self._state = info.get('direct', {}).get('time')
+                    self._state = int(info.get('direct', {}).get('time'))
```

Comment:
```suggestion
                    self._state = info.get('direct', {}).get('time')
```
States are always strings